test/Microsoft.SqlTools.ServiceLayer.UnitTests/Connection/ConnectionServiceTests.cs

```diff
@@ -1219,15 +1219,5 @@ public void ConnectParamsAreInvalidIfConnectionIsNull()
 
             Assert.NotEmpty(connectionResult.ConnectionId);
         }
-
-        [Fact]
-        public async Task ConnectionWithBadConnectionStringFails()
```

Comment:
That functionality is already tested by BuildConnectionStringWithInvalidAuthType, which is why I removed this test.